src/app/components/patterns/DemoCard.tsx

```diff
@@ -78,27 +68,29 @@ const useStyles = makeStyles((theme: Theme) =>
             color: theme.palette.text.primary,
         },
         demoCard: {
-            width: theme.spacing(35),
+            width: theme.spacing(38),
```

Comment:
Demo Card width doesn't line up with the image widths at max size:
![image](https://user-images.githubusercontent.com/29152776/80964208-709fd300-8dde-11ea-95bb-8491cf466562.png)

Reply:
Under the current 766 / 798 px width, there is an obvious difference in the card-image width. Hopefully it wouldn't make you too uncomfortable now.